src/Microsoft.Data.SqlClient/src/Microsoft/Data/OperationAbortedException.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace Microsoft.Data
 {
```

Comment:
For function `internal static OperationAbortedException Aborted(Exception inner)`, can we keep the ` ADP.TraceExceptionAsReturnValue(e);` to have event source trace here?

Reply:
Done.